etl/steps/data/grapher/countries/2023-10-01/isd.meta.yml

```diff
@@ -3,5 +3,5 @@ tables:
     variables:
       is_present:
         title: Sovereign state
-        description_short: Whether the country is present in the ISD countries dataset.
+        description_short: The country is identified as a sovereign state by the ISD dataset.
```

Comment:
again here - e.g., `A country is recognized as a sovereign state by the ISD if it has a minimum population of 10,000, significant internal control over its territory, and is externally acknowledged by peers with autonomy in its foreign relations.`

Reply:
Leaving as-is, as Bastian proposed this definition.